notes_app/notes/api.py

```diff
@@ -39,6 +39,44 @@ def delete_detail(self, object_list, bundle):
         raise Unauthorized("Deletes not allowed")
 
 
+# Custom User Authorization
+class UserObjectsOnlyAuthorization(Authorization):
+    def read_list(self, object_list, bundle):
+        # This assumes a ``QuerySet`` from ``ModelResource``.
+        return object_list.filter(user=bundle.request.user)
```

Comment:
this makes no sense - on list request user will get only himself. How he would be able to share notes? Please fix the logic here. User can read list of users